src/Opserver.Web/Controllers/ExceptionsController.cs

```diff
@@ -39,6 +43,16 @@ public override void OnActionExecuting(ActionExecutingContext context)
             CurrentId = GetParam("id").HasValue() && Guid.TryParse(GetParam("id"), out var guid) ? guid : (Guid?)null;
             CurrentSimilarId = GetParam("similar").HasValue() && Guid.TryParse(GetParam("similar"), out var similarGuid) ? similarGuid : (Guid?)null;
             Enum.TryParse(GetParam("sort"), out CurrentSort);
+            CurrentUrl = GetParam("url")?.Trim();
+            CurrentHost = GetParam("host")?.Trim();
+            if (GetDate("startDate") is DateTime startDate)
```

Comment:
I've renamed "start" and "end" to "startDate" and "endDate". The original names are not used on the codebase and I wanted to align these values with their property name (hope they are not use by a script that fetches info from opserver). I've also moved them to were the rest of properties are parsed.